tests/by-util/test_cp.rs

```diff
@@ -1465,6 +1465,19 @@ fn test_cp_archive_on_nonexistent_file() {
 
 #[test]
 #[cfg(unix)]
+fn test_copy_symlink_with_no_dereference_and_attributes_only() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.touch("file");
+    at.symlink_file("file", "sym");
+    ucmd.arg("-d")
+        .arg("--attributes-only")
+        .arg("sym")
+        .arg("sym2")
+        .succeeds();
+    assert!(at.is_symlink("sym2"));
+}
+
+#[test]
```

Comment:
```suggestion
#[test]
#[cfg(unix)]
```
since `mkfifo` doesn't exist on Windows